src/components/FilterSearch.tsx

```diff
@@ -77,23 +78,37 @@ export function FilterSearch({
     return { ...searchField, fetchEntities: false };
   });
   const cssClasses = useComposedCssClasses(builtInCssClasses, customCssClasses);
+  const [currentFilter, setCurrentFilter] = useState<Filter>();
+  const [query, setQuery] = useState<string>();
+  const filters = useSearchState(state => state.filters.static);
 
   const [
     filterSearchResponse,
-    executeFilterSearch
+    executeFilterSearch,
+    clearFilterSearchResponse
   ] = useSynchronizedRequest<string, FilterSearchResponse>(
-    inputValue => searchActions.executeFilterSearch(inputValue ?? '', sectioned, searchParamFields),
+    inputValue => {
+      setQuery(inputValue);
```

Comment:
Small nit: Could we call this `filterQuery` and `setFilterQuery`? I only ask because at first glance, this looks like we're calling `AnswersActions.setQuery`.

Reply:
sure thing